portfolio/components/ui/Figures/DiffVisualization/index.tsx

```
@@ -0,0 +1,755 @@
+
+  const fetchMoreReceipts = useCallback(async () => {
+    if (isFetchingRef.current || isPoolExhausted) return;
+    isFetchingRef.current = true;
+
+    try {
+      const response = await api.fetchLabelEvaluatorDiff(20);
+      if (response && response.receipts) {
+        const newReceipts = response.receipts.filter(
+          (r) => !seenReceiptKeys.current.has(`${r.image_id}_${r.receipt_id}`)
+        );
+        newReceipts.forEach((r) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)